<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BotWidget</class>
 <widget class="QWidget" name="BotWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>650</width>
    <height>650</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <property name="leftMargin">
    <number>3</number>
   </property>
   <property name="topMargin">
    <number>3</number>
   </property>
   <property name="rightMargin">
    <number>3</number>
   </property>
   <property name="bottomMargin">
    <number>3</number>
   </property>
   <property name="horizontalSpacing">
    <number>0</number>
   </property>
   <property name="verticalSpacing">
    <number>3</number>
   </property>
   <item row="0" column="0" colspan="2">
    <widget class="HeaderWidget" name="W_HEADER" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>180</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>180</height>
      </size>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QTabWidget" name="tabWidget">
     <property name="minimumSize">
      <size>
       <width>600</width>
       <height>400</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>132456</width>
       <height>123123</height>
      </size>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Overall</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_5">
       <item row="2" column="1">
        <widget class="MapWidget" name="W_MAP" native="true"/>
       </item>
       <item row="4" column="1">
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="0" column="1">
        <widget class="QLabel" name="L_MAP_NAME">
         <property name="text">
          <string/>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="0" column="0" rowspan="5">
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QWidget" name="widget" native="true">
           <layout class="QGridLayout" name="gridLayout_27">
            <item row="1" column="0">
             <widget class="UsersWidget" name="W_USERS" native="true"/>
            </item>
            <item row="0" column="0">
             <widget class="QLabel" name="label">
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>12</height>
               </size>
              </property>
              <property name="text">
               <string>Users (MAP)</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </item>
       <item row="0" column="2" rowspan="5">
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QPushButton" name="PB_CLEAR">
             <property name="text">
              <string>Clear</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_11">
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>15</height>
              </size>
             </property>
             <property name="text">
              <string>CHAT</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignBottom|Qt::AlignRight|Qt::AlignTrailing</set>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QTextBrowser" name="TB_CHAT">
           <property name="minimumSize">
            <size>
             <width>120</width>
             <height>20</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>Arial Narrow</family>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true"/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="LE_CHAT">
           <property name="minimumSize">
            <size>
             <width>120</width>
             <height>20</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_10">
      <attribute name="title">
       <string>Manage?</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_16">
       <item row="0" column="0">
        <widget class="QTabWidget" name="tabWidget_3">
         <property name="currentIndex">
          <number>0</number>
         </property>
         <widget class="QWidget" name="tab_11">
          <attribute name="title">
           <string>Pathing</string>
          </attribute>
          <layout class="QGridLayout" name="gridLayout">
           <item row="0" column="0">
            <widget class="PathingWidget" name="W_PATHING" native="true"/>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="tab_13">
          <attribute name="title">
           <string>Skills</string>
          </attribute>
          <layout class="QGridLayout" name="gridLayout_21">
           <item row="0" column="0">
            <widget class="SkillsWidget" name="W_SKILLS" native="true"/>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="tab_12">
          <attribute name="title">
           <string>Items</string>
          </attribute>
          <layout class="QGridLayout" name="gridLayout_15">
           <item row="0" column="0">
            <widget class="ItemUseWidget" name="W_ITEMS" native="true"/>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="tab_14">
          <attribute name="title">
           <string>Behaviour</string>
          </attribute>
          <layout class="QGridLayout" name="gridLayout_6">
           <item row="0" column="0">
            <widget class="BehaviourWidget" name="W_BEHAVIOUR" native="true"/>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="tab_15">
          <attribute name="title">
           <string>Monsters</string>
          </attribute>
          <layout class="QGridLayout" name="gridLayout_12">
           <item row="0" column="0">
            <widget class="MonsterWidget" name="W_MONSTERS" native="true"/>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="tab_16">
          <attribute name="title">
           <string>Drops</string>
          </attribute>
          <layout class="QGridLayout" name="gridLayout_13">
           <item row="0" column="0">
            <widget class="DropWidget" name="W_DROPS" native="true"/>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="tab_9">
          <attribute name="title">
           <string>Inventory</string>
          </attribute>
          <layout class="QGridLayout" name="gridLayout_4">
           <item row="0" column="0">
            <widget class="InventoryWidget" name="W_INVENTORY" native="true"/>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_5">
      <attribute name="title">
       <string>Security</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_28">
       <item row="0" column="0">
        <widget class="SecurityWidget" name="W_SECURITY" native="true"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Statistics</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_25">
       <item row="0" column="0">
        <widget class="QGroupBox" name="groupBox_12">
         <property name="title">
          <string>EXP earned</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_24">
          <item row="1" column="1">
           <widget class="QLabel" name="L_EXPHERO_PRCT">
            <property name="text">
             <string>Value Heroic exp</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QLabel" name="L_EXPJOB_PRCT">
            <property name="text">
             <string>Value Job exp</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_41">
            <property name="text">
             <string>Job</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLabel" name="L_EXP_PRCT">
            <property name="text">
             <string>Value Normal exp</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_38">
            <property name="text">
             <string>Heroic</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="label_36">
            <property name="text">
             <string>Normal</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QGroupBox" name="groupBox_13">
         <property name="title">
          <string>Damage</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_29">
          <item row="3" column="0">
           <widget class="QLabel" name="label_70">
            <property name="text">
             <string>Total</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="label_69">
            <property name="text">
             <string>Maximum</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_57">
            <property name="text">
             <string>Min</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_22">
            <property name="text">
             <string>Average</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLabel" name="L_DMG_MAX">
            <property name="text">
             <string>MAX_D</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLabel" name="L_DMG_AVG">
            <property name="text">
             <string>AVG_D</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QLabel" name="L_DMG_MIN">
            <property name="text">
             <string>MIN_D</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QLabel" name="L_DMG_SUM">
            <property name="text">
             <string>SUM_D</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="1" column="0" colspan="2">
        <layout class="QGridLayout" name="gridLayout_23">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item row="3" column="0">
          <layout class="QHBoxLayout" name="horizontalLayout_15">
           <property name="bottomMargin">
            <number>20</number>
           </property>
           <item>
            <widget class="QLabel" name="L_HORLOGE_IMG">
             <property name="minimumSize">
              <size>
               <width>30</width>
               <height>30</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>30</width>
               <height>30</height>
              </size>
             </property>
             <property name="text">
              <string>HORLOGE</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="L_TOTAL_TIME">
             <property name="minimumSize">
              <size>
               <width>50</width>
               <height>20</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>50</width>
               <height>20</height>
              </size>
             </property>
             <property name="text">
              <string>Time spent</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_6">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QLabel" name="label_66">
             <property name="text">
              <string>RANK </string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="L_RANK">
             <property name="text">
              <string>TextLabel</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="2" column="0">
          <spacer name="verticalSpacer_9">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="1" column="0">
          <layout class="QGridLayout" name="gridLayout_26">
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item row="1" column="2">
            <widget class="QTableWidget" name="TW_STAT_USED"/>
           </item>
           <item row="1" column="1">
            <widget class="QTableWidget" name="TW_STAT_DROPS"/>
           </item>
           <item row="1" column="0">
            <widget class="QTableWidget" name="TW_STAT_MONSTERS"/>
           </item>
           <item row="0" column="0">
            <widget class="QLabel" name="label_61">
             <property name="text">
              <string>Monsters killed</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLabel" name="label_62">
             <property name="text">
              <string>Drops</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item row="0" column="2">
            <widget class="QLabel" name="label_63">
             <property name="text">
              <string>Items used</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>HeaderWidget</class>
   <extends>QWidget</extends>
   <header>widgets/headerwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>InventoryWidget</class>
   <extends>QWidget</extends>
   <header>widgets/inventorywidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ItemUseWidget</class>
   <extends>QWidget</extends>
   <header>widgets/itemusewidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>SecurityWidget</class>
   <extends>QWidget</extends>
   <header>widgets/securitywidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>UsersWidget</class>
   <extends>QWidget</extends>
   <header>widgets/userswidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>MapWidget</class>
   <extends>QWidget</extends>
   <header>widgets/mapwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>SkillsWidget</class>
   <extends>QWidget</extends>
   <header>widgets/skillswidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>BehaviourWidget</class>
   <extends>QWidget</extends>
   <header>widgets/behaviourwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>MonsterWidget</class>
   <extends>QWidget</extends>
   <header>widgets/monsterwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>DropWidget</class>
   <extends>QWidget</extends>
   <header>widgets/dropwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>PathingWidget</class>
   <extends>QWidget</extends>
   <header>widgets/pathingwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
